src/utils/url_utils.ts

```diff
@@ -0,0 +1,20 @@
+const S3_URL_PREFIX = "s3://";
+const GCS_URL_PREFIX = "gs://";
+
+/**
+ * Remaps non-standard URIs (e.g. S3 (s3://) or Google Cloud Storage (gs://)) to
+ * a standard HTTPS URL.
+ */
+export function remapUri(url: string): string {
+  url = url.trim();
```

Comment:
Potentially iffy to be both modifying and returning `url`? Consider assigning a new variable for the output